### PR TITLE
added unit tests 

### PR DESCRIPTION
**Pull request by aryangupta701**
added unit tests and drivers to browsers in integration tests.

### CODE REVIEW

File: __mocks__/webextension-polyfill.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related source files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2022 The ZAP Development Team
```
Comment (kingthorin) [inline __mocks__/webextension-polyfill.ts:6]: 2023

File: test/ContentScript/unitTests.test.ts [hunk 2]
```diff
@@ -0,0 +1,268 @@
+/**
+ * @jest-environment jsdom
+ */
+/*
+ * Zed Attack Proxy (ZAP) and its related source files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2022 The ZAP Development Team
```
Comment (kingthorin) [inline test/ContentScript/unitTests.test.ts:9]: 2023

File: test/drivers/ChromeDriver.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related source files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2022 The ZAP Development Team
```
Comment (kingthorin) [inline test/drivers/ChromeDriver.ts:6]: 2023

File: source/Background/index.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@
  *
  * ZAP is an HTTP/HTTPS proxy for assessing web application security.
  *
- * Copyright 2022 The ZAP Development Team
+ * Copyright 2023 The ZAP Development Team
```
Comment (psiinon) [inline source/Background/index.ts:6]: You dont have to go back an update old Copyright years. Doesnt hurt in the case, so just an FYI. We only need to put the current year in new files.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**